posthog/hogql/property.py

```diff
@@ -156,54 +160,57 @@ def _handle_bool_values(value: ValueT, field: ast.Field, property: Property, tea
     return value
 
 
-def _field_to_compare_op(
```

Comment:
I might pull these changes into a separate PR and get approval from Product Analytics